libs/positions/src/lib/positions.tsx

```diff
@@ -82,8 +82,8 @@ export const Positions = memo(
     };
     return (
       <AsyncRenderer loading={loading} error={error} data={data}>
-        <div className="flex justify-between items-center p-3 pb-1">
-          <h4 className="text-m">
+        <div className="flex justify-between items-center px-4 pt-3 pb-1">
```

Comment:
This was trying to reclaim some space in the positions panel so it can show more positions, but I've got another branch where I'm still fiddling with that.

Reply:
px-4 is required to align the text nicely otherwise it looked a little off.  Top and bottom are the same as before just written slightly clearer